### DISCUSSION TITLE
Identifier in the Add FX list to differentiate diffent types of FX

### DISCUSSION THREAD

**Discussion by WolfInABowl** [opentoonz/opentoonz#6473] [Ideas]
Currently, there is no way to tell if an FX will create a new Level e.g. `TargetSpot` or any of the gradient nodes, or if it will only exist within the FX Schematic.
It would make sense for an icon/color identifier to be used to communicate the type of FX to the user.

**Comment by DarrenTAnims** [comment]
Sounds sensible to me.

**Comment by artisteacher** [comment]
Some sort of indicator would be helpful (also for Insert FX and Replace FX) but could get very busy visually. If OT had Tahoma's status bar, that would be a great place to give more information about the FX without adding visual clutter. 

Personally, the first thing I would do to improve adding/inserting/replacing FX would be adding a search field.

**Comment by gab3d** [comment]
there already is a search field in FX Browser, or you mean something different?

**Comment by RodneyBaker** [comment]
Just to clarify, the search field is available in one location related to the FX Browser... via New FX in xsheet/timeline... as it was only added to that process.  When accessing via other means it won't appear.  Here we see the search field via New FX:

![image](https://user-images.githubusercontent.com/2576432/114469074-17295200-9bb2-11eb-8c83-6240a9cdf23d.png)

And here we don't directly via schematic using Add FX:

![image](https://user-images.githubusercontent.com/2576432/114469378-a20a4c80-9bb2-11eb-8ead-f21429fe17f8.png)

**Comment by artisteacher** [comment]
Yeah, I always forget about the FX browser. I always use the context menu options.
